src/Exceptions/CouldNotTakeBrowsershot.php

```diff
@@ -15,4 +15,19 @@ public static function outputFileDidNotHaveAnExtension(string $path)
     {
         return new static("The given path `{$path}` did not contain an extension. Please append an extension.");
     }
+
+    public static function postParamsArrayIsNotAssoc()
+    {
+        return new static('The given postParams array is not associative.');
```

Comment:
Tack on "Make sure that every element has a key." to this message.

Reply:
Removed this exception as redundant.